### ISSUE TITLE
Nested Controllers

### ISSUE THREAD

**Issue by HamidBehnam** [angelozerr/angularjs-eclipse#142]
Hi there,

I tried to insert nested controllers but I got this error: 

Multiple annotations found at this line:
	- Undefined attribute name (ng-controller).
	- Cannot find controller with name SecondController

Code:

![capture](https://cloud.githubusercontent.com/assets/5709332/6046133/577e3054-ac54-11e4-839e-f023fded85be.PNG)

**Comment by angelozerr** [comment]
Please provide a basic sample (HTML + JS) with your case. Thanks.

**Comment by HamidBehnam** [comment]
http://plnkr.co/edit/4vsbGawedMnKfh0mj74o

**Comment by HamidBehnam** [comment]
If you try to change the value of ng-app and save the file and then roll back the change and save the file again, the error will be gone.

**Comment by angelozerr** [comment]
It works for me?

**Comment by HamidBehnam** [comment]
Yes the page works, and there isn't any problem with angularjs's functionality, but without change instruction that I wrote, the error message will be shown. If you have a team viewer installed I can show  you where's the problem.

**Comment by angelozerr** [comment]
You mean taht you must save file in order to that error disapear. If it that it's because you have "Validation" in your Project Builder.

The error disappears when you save. It's the same behaviour with HTML validation.

**Comment by HamidBehnam** [comment]
Validation (in Tern menu) is disabled. 
I made a mistake about saving the file even there's no need to save the file, to disappear the error message I have to change the value of ng-app, and then when I roll back the value of the ng-app, the error message disappears.

**Comment by angelozerr** [comment]
> Validation (in Tern menu) is disabled. 

This Validation is only for JavaScript files if you select "lint" tern module.

I mean "Validation" in Builder menu. This Validation comes from Eclipse WTP

**Comment by HamidBehnam** [comment]
I'm not gonna turn eclipse default validation off, you made an awesome plugin to use angularjs auto complete in eclipse, but why you don't want to figure this issue out? 

Is this scenario normal?
"An error message appears, I add a character to the ng-app then I remove that character (without saving the file) and error disappear."  

it sounds a little bit strange. doesn't it?

**Comment by angelozerr** [comment]
> but why you don't want to figure this issue out?

I would like to help you, but I cannot reproduce your problem. Perhaps if you add some screenshot with your scenario, I could reproduce your problem?

**Comment by HamidBehnam** [comment]
How can I send you a private message? I'd like to give you a Teamviewer session's password to show you the problem.

**Comment by angelozerr** [comment]
I'm sorry I have no time for that. Please attach screenshots with your problem. Thanks!

**Comment by HamidBehnam** [comment]
I'll not take about 20 screenshots since I've explained the issue very well, and the brief summary is: your plugin has some difficulties to recognize nested controllers. 

Any way I've found some tricky ways to manage that (such as moving the nested controller one level up and then one level down !), I hope other users find some tricky ways too.

I'm not going to continue this thread since I think I had to report the problem and I've done, by the way I appreciate your efforts to bring this functionality (angularjs auto completion) to eclipse users.
